applications/openshift/master/file_groupowner_ovs_sys_id_conf/rule.yml

```diff
@@ -0,0 +1,34 @@
+documentation_complete: true
+
+prodtype: ocp4
+
+title: 'Verify Grop Who Owns The Open vSwitch Persistent System ID'
+
+description: |-
+    {{{ describe_file_group_owner(file="/etc/openvswitch/system-id.conf", group="800") }}}
+
+rationale: |-
+    CNI (Container Network Interface) files consist of a specification and libraries for
+    writing plugins to configure network interfaces in Linux containers, along with a number
+    of supported plugins. Allowing writeable access to the files could allow an attacker to modify
+    the networking configuration potentially adding a rouge network connection.
+
+severity: medium
+
+#identifiers:
+#    cce@ocp4:
+
+references:
+    cis: 1.1.9
+
+ocil_clause: '{{{ ocil_clause_file_group_owner(file="/etc/openvswitch/system-id.conf", group="800") }}}'
+
+ocil: |-
+    {{{ ocil_file_group_owner(file="/etc/openvswitch/system-id.conf", group="owner") }}}
```

Comment:
is `group="owner"` here and in the other occurences correct?

Reply:
no... it was a typo in my script